app/src/main/java/org/fossasia/openevent/app/contract/view/LoginView.java

```diff
@@ -0,0 +1,11 @@
+package org.fossasia.openevent.app.contract.view;
+
+public interface LoginView {
+
+    void showProgressBar(boolean show);
```

Comment:
Please see the `View` interface here

https://medium.com/@cervonefrancesco/model-view-presenter-android-guidelines-94970b430ddf